packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -94,6 +83,22 @@ export function activate(context: vscode.ExtensionContext) {
     debugConfigProvider,
     checkpointsView
   );
+
+  await checkpointService.activateRequestService();
+
+  // Load the breakpoint line info stuff right here. Need to await this, otherwise
+  // we can (and will if they're stored) get breakpoint events before the line
+  // breakpoint information is fully loaded.
+  if (!BreakpointUtil.getInstance().hasLineNumberMapping()) {
+    console.log('in activate, getting line breakpoint info');
+    await retrieveLineBreakpointInfo();
+  }
+
+  // This particular event handler will get called whether or not there
+  // is an active debug session. It is necessary for checkpoints to able to
+  // be created without an active debug session since the information from
+  // them will potentially be used for the reply-debugger session.
+  vscode.debug.onDidChangeBreakpoints(processBreakpointChangedForCheckpoints);
```

Comment:
Can you make this a `Disposable`? See `context.subscriptions` above.

Reply:
My fault for not telling you about this Jim. I think when we talked I mentioned that you didn't need to push this as a disposable but it should be since we want this listener to be thrown away when the extension is deactivated.